buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -46,13 +50,25 @@ public void beforeStart() {
                 entry -> entry.getKey().toString().substring(CUSTOM_SETTINGS_PREFIX.length()),
                 entry -> entry.getValue().toString()
             ));
+        Path baseDataPath = null;
+        String dataPathSetting = System.getProperty(DATA_DIR_SETTING);
+        if (dataPathSetting != null) {
+            baseDataPath = Paths.get(dataPathSetting).toAbsolutePath();
+        }
         for (ElasticsearchCluster cluster : getClusters()) {
             cluster.getFirstNode().setHttpPort(String.valueOf(httpPort));
             httpPort++;
             cluster.getFirstNode().setTransportPort(String.valueOf(transportPort));
             transportPort++;
+            Path clusterDataPath = null;
```

Comment:
Why all this complexity? Why not just set the data path to exactly what folks provide, otherwise go with the default? Creating subdirectories under the provided path is a bit unintuitive and won't give the expected result if folks have an existing data dir that they want to test against.

Reply:
I pushed 39198fd to simplify the usage of the dir when we only have a single node.